rpc/websocket.go

```diff
@@ -50,10 +50,11 @@ var wsBufferPool = new(sync.Pool)
 // To allow connections with any origin, pass "*".
 func (s *Server) WebsocketHandler(allowedOrigins []string) http.Handler {
```

Comment:
if we change to `WebsocketHandler(enableCompression bool, allowedOrigins []string) http.Handler {` we can acheive only compress between coordinator&rollers

Reply:
Use `SetCompressionLevel` function can only let it work.